README.md

```diff
@@ -1 +1,8 @@
 # Socket.io Chat App with Next.js
+
+A simple chat app built with Next.js (for client) and Socket.io (for server).
```

Comment:
Add these sections:
- how-to run locally
- how to build images
- how to use Terraform
- how to use K8s